curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6565bd4265158360de8e2ae7.md

```diff
@@ -7,8 +7,7 @@ dashedName: step-59
 
 # --description--
 
-Turn the expression inside your `for` loop into an `if` statement. Use the expression you wrote in the previous step as the `if` condition.
-
```

Comment:
please do not remove the empty line, we still want to have two separete paragraphs